alloc.h

```diff
@@ -1,8 +1,17 @@
 #ifndef ALLOC_H
 #define ALLOC_H
 
-extern /*@null@*//*@out@*/char *alloc();
-extern void alloc_free();
-extern int alloc_re();
+#include <stdlib.h>
+
+#define alloc(n) malloc(n)
+#define alloc_free(x) free(x)
+static inline int alloc_re(char **x, unsigned int m, unsigned int n)
```

Comment:
I missed that `x` is a pointer passed as a reference, so it did segfault on my iteration, so good catch.

Reply:
You are not the only one ;)